packages/functional-tests/src/tests/physics-bounce-test.ts

```diff
@@ -7,21 +7,21 @@ import * as MRE from '@microsoft/mixed-reality-extension-sdk';
 
 import { Test } from '../test';
 
-const defaultBallColor = MRE.Color3.FromInts(220, 150, 150);
-
 export default class PhysicsBounceTest extends Test {
 	public expectedResultDescription = "Balls and boxes hit the ground and bounce.";
 	private assets: MRE.AssetContainer;
 	private interval: NodeJS.Timeout;
-	private ballboxMat: MRE.Material;
+	private materials: MRE.Material[] = [];
```

Comment:
Isn't this redundant with `this.assets.materials`?

Reply:
Not exactly. If there were any more materials defined in the container then this would the subset applicable to the spawned physics objects. I could rename it `palette`.